css/styles.css

```diff
@@ -625,95 +913,348 @@ figure {
     font-family: "Bakbak One", Arial, sans-serif;
     font-size: 44px;
     font-weight: 400;
-    line-height: 56px;
+    line-height: 1.27;
     text-align: left;
     color: #FFFFFF;
 }
 
-.catalog-input {
-    line-height: 26px;
-    text-align: left;
-    background-color: #141B22;
-    border: none;   
+.hero-list {
+    display: flex;
+    align-items: center;
+    gap: 8px;
 }
 
-.catalog-input ::placeholder {
-    color: #888B8E;
+.vertical-line {
+    border: 1px solid #888B8E;
+    background-color: #888B8E;
+    height: 16px;
 }
 
-/* -------------- catalog filter --------------*/
-
-.catalog-filter {
-    background-color: #141B22;
+.search-form {
+    position: relative;
 }
 
-.catalog-filter-option {
-    font-family: Poppins, Arial, sans-serif;
+.catalog-input {
+    font-family: Poppins;
     font-size: 16px;
     font-weight: 400;
-    line-height: 26px;
-    text-align: left;
-    color: #F7F7F7;
+    line-height: 1.63;
     background-color: #141B22;
-    border: 1px solid #41454A;
+    width: 280px;
+    padding: 13px 20px 9px 24px;
+    border: none;  
+    outline: none;
 }
 
-.catalog-filter-reset-btn {
-    font-family: "Bakbak One", Arial, sans-serif;
-    font-size: 14px;
-    font-weight: 400;
-    line-height: 22px;
-    text-align: left;
-    color: #21E786;
-    background-color: #141B22;
-    border: none;
+.catalog-input::placeholder {
+    font-size: 16px;
+    line-height: 1.63;
+    color: #888B8E;
 }
 
-/* -------------- catalog range --------------*/
-
-.catalog-range-item {
-    background-color: #FFFFFF0D;
+.catalog-input:focus {
+    outline: none;
+    color: #C2C3C5;
 }
 
-/*.catalog-range-img {
-       
+.subscribe-input:focus {
+    outline: none;
+    color: #C2C3C5;
+    background-color: #C2C3C526;
 }
-*/
-.catalog-range-title {
-    font-family: "Bakbak One", Arial, sans-serif;
-    font-size: 20px;
-    line-height: 28px;
-    /*text-align: center;*/
-    text-transform: uppercase;
-    color: #FFFFFF;
+
+.buttonn-search {
+    position: absolute;
+    display: block;
+    top: 13px;
+    right: 20px;
+    background-color: transparent;
+    border: none;
 }
 
-/* -------------- FOOTER SECTION --------------*/
+/* -------------- catalog filter --------------*/
 
-/* -------------- footer-catalog-navigation --------------*/
+.catalog {
+    display: flex;
+    gap: 80px;
+    margin-bottom: 100px;
+}
 
-.footer-catalog-logo-link {
-    font-family: "Bakbak One", Arial, sans-serif;
-    font-size: 30px;
-    line-height: 42px;
-    text-align: left;
-    text-transform: uppercase;
-    color: #FFFFFF;
+/* -------------- sidebar --------------*/
+
+.sidebar {
+    width: 280px;
 }
 
-.footer-catalog-logo-text {
+.sidebar-form {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 12px;
+}
+
+.sidebar-list {
+    background: #141B22;
+    width: 100%;
+}
+
+.sidebar-item {
```

Comment:
Please add `cursor: pointer` for this selector - user should be able to see that is clickable element